cmd/containerboot/main.go

```diff
@@ -306,81 +313,102 @@ authLoop:
 			log.Fatalf("error creating new netfilter runner: %v", err)
 		}
 	}
+	notifyChan := make(chan ipn.Notify)
+	errChan := make(chan error)
+	go func() {
+		for {
+			n, err := w.Next()
+			if err != nil {
+				errChan <- err
```

Comment:
I know currently it does not matter (since we are exiting the process anyway), but perhaps we should return from this loop on error, and on `ctx.Done()`?

Reply:
I've added a break for the error case. That should cover `ctx.Done()` too since we pass the same context when creating the watcher which should then return an error when context is done
https://github.com/tailscale/tailscale/blob/v1.52.1/client/tailscale/localclient.go#L1350-L1351